### ISSUE TITLE
sorting when using ajax loading

### ISSUE THREAD

**Issue by TomBruyneel** [olifolkerd/tabulator#681]
It seems that there is a change in 3.3 with causes an issue when sorting with ajax sorting enabled, the column headers are not updated. 

```js
if (!self.table.options.ajaxSorting) {

        self.sortList.forEach(function (item, i) {

          if (item.column && item.column.extensions.sort) {

            //if no sorter has been defined, take a guess


            if (!item.column.extensions.sort.sorter) {

              item.column.extensions.sort.sorter = self.findSorter(item.column);
            }

            self._sortItem(item.column, item.dir, self.sortList, i);
          }

          self.setColumnHeader(item.column, item.dir);
        });
      }
```

This causes the aria-sort attribute to be always none, which effectively breaks sorting. Any idea what we are doing wrong?

**Comment by cschnelting** [comment]
I have the same Error on Tabulator v3.3. When AjaxSorting is enabled, the Sorter-Arrows are not updated.

**Comment by olifolkerd** [comment]
This should be fixed in version 3.3.1, which version are you running?

**Comment by mitchelljkotler** [comment]
I am using 3.3.1 and also see this issue.  In addition - I am using remote pagination, and with ajaxSorting set to true or false, the ajax request includes a sorters parameter that is the string '[object Object]' instead of a properly serialized sorter object.

**Comment by TomBruyneel** [comment]
I'm using version 3.3.1. The snippet above is from that version, as you see the setcolumnheader call is within the if block and is not called anywhere else as far as I can see.

**Comment by mitchelljkotler** [comment]
@TomBruyneel Do you see the same issue as I do with regards to the sorters parameter not being serialized correctly?

**Comment by cschnelting** [comment]
I have downloaded the newest Tabulator.js and now it works correct. Thanks!

**Comment by olifolkerd** [comment]
how are you trying to sort, by clicking on the column headers or using the ***setSort*** function?

which browser are you using?

**Comment by mitchelljkotler** [comment]
I'm trying to sort by clicking the header.  I am using Firefox.

**Comment by mitchelljkotler** [comment]
I resolved the [ibject Object] issue by passing traditional: false to the ajaxConfig.  I still see the original error @TomBruyneel is referring to.  The fix must not have made it into the npm 3.3.1 release.

**Comment by TomBruyneel** [comment]
Also trying to sort while clicking on the header, the issue is present in all browsers.

**Comment by olifolkerd** [comment]
Hey Guys,

im really struggling to replicate this behaviour at all, could one of you post a VERY SIMPLE table constructor with a couple of columns that replicates this issue, whenever i click a header the ajax sort dir changes and the aria tag changes.

i need to know the browser you are using, exact step by step of how to replicate it and ideally a sample ajax response

A lot of you are having the problem so i trust that there must be an issue but i just can replicate it.

Cheers

Oli

**Comment by olifolkerd** [comment]
I think I have gotten to the bottom of this, I pushed a fix to it to master back in October but completely forgot to do an official release for it, I have just released patch fix 3.3.2 which includes the fix for this,

Let me know if you are still having issues after the update.

Cheers

Oli
